### PR TITLE
updated README.md with some basic grammar

### PR DESCRIPTION
**Pull request by nikunjdk**

### CODE REVIEW

**Review by vinaysomawat — APPROVED** [review]
You have used grammerly tool to correct grammer mistakes but sometimes it changes the context of sentences so read it before changing it.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![alt text](./aarogya.png "AarogyaSetu Logo")
 
-Aarogya Setu is a mobile application developed by the Government of India to connect essential health services with the people of India in our combined fight against COVID-19. The App is aimed at augmenting the initiatives of the Government of India, particularly the Department of Health, in proactively reaching out to and informing the users of the app regarding risks, best practices and relevant advisories pertaining to the containment of COVID-19.
+Aarogya Setu is a mobile application developed by the Government of India to connect essential health services with the people of India in our combined fight against COVID-19. The App is aimed at augmenting the initiatives of the Government of India, particularly the Department of Health, in proactively reaching out to and informing the users of the app regarding risks, best practices and relevant advisories on the containment of COVID-19.
```
Comment (tusharojha) [inline README.md:5]: best practices, and relevant advisories on the containment of COVID-19.
add a comma after "best practices".

**Review by tusharojha — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nikunjdk) [reply README.md:5]: Since "relevant advisories" is the last feature of the app in this sentence I believe that there is no need for a comma.

**Comment by nikunjdk** [comment]
> You have used Grammarly tool to correct grammar mistakes but sometimes it changes the context of sentences so read it before changing it.

Can you please mention where the context of the sentence has been changed because I have only accepted those suggestions where I believe the context has not been changed.

**Pull request closed without merging**